integration/cursor/getmore_test.go

```diff
@@ -627,7 +627,8 @@ func TestGetMoreCommandConnection(t *testing.T) {
 }
 
 func TestGetMoreCommandMaxTimeMSErrors(t *testing.T) {
-	// do not run tests in parallel to avoid using too many backend connections
```

Comment:
The context related to this comment is now lost and I'm not seeing issue with parallel tests.

Reply:
It was about PostgreSQL running of `max_connections` while running a lot of tests in parallel

Reply:
Ah I see, so it run out of connections. Do we still want prevent the test from running parallel?

Reply:
Let's remove it if it works, but we might need to add it back